docker/docker-compose.dev.yml

```diff
@@ -32,7 +32,14 @@ services:
       FIREBASE_CLIENT_EMAIL:
       FIREBASE_PRIVATE_KEY:
       FIREBASE_PREFIX:
+      MAILER_TRANSPORT:
+      MAILER_HOST:
+      MAILER_USER:
+      MAILER_PASSWORD:
+      MAILER_AUTH_MODE:
+      MAILER_PORT:
       CREATE_ACCOUNT_FEATURE:
+      USE_MAIL_FEATURE:
```

Comment:
Should this be configured as a "Feature Toggle" the same way the other features (commenting and create account) are?

Reply:
Added the config in the feature toggle config section